php/php.phpdoc/src/org/netbeans/modules/php/phpdoc/PhpDocScript.java

```diff
@@ -145,13 +149,24 @@ private String sanitizePath(String path) {
         return path;
     }
 
-    private List<String> getParameters(String sanitizedPhpDocTarget, PhpModule phpModule) {
+    private List<String> getAllParameters(String sanitizedPhpDocTarget, PhpModule phpModule) {
+        List<String> params = new ArrayList<>(getDefaultParameters(sanitizedPhpDocTarget, phpModule));
+        if (PhpDocPreferences.isConfigurationEnabled(phpModule)) {
+            String configurationPath = PhpDocPreferences.getPhpDocConfigurationPath(phpModule);
+            if (!StringUtils.isEmpty(configurationPath)) {
+                params.add(PARAM_CONFIG);
+                params.add(sanitizePath(PhpDocPreferences.getPhpDocConfigurationPath(phpModule)));
```

Comment:
If the path is incorrect, PhpDoc itself will complain, I guess, so we don't need to run the validator here. Or do we? :thinking: 

AFAIR we validate project properties before each run and if they are incorrect, the Project Properties dialog is opened.

@junichi11 @KacerCZ What do you think, guys?

Reply:
Properties are opened only if target directory for generated documentation is not valid.
`sanitizePath()` only replaces backslashes on Windows with slashes.

Reply:
> Properties are opened only if target directory for generated documentation is not valid.

So, should not it be opened also if the enabled configuration file is invalid, typically it does not exist?

Reply:
I'll improve it.

Reply:
Thank you.

Reply:
Fixed.
![netbeans-5244-phpdocumentor-invalid-settings](https://user-images.githubusercontent.com/738383/104852801-4b3d0000-5940-11eb-9cf9-4cbff474cb1b.gif)

Reply:
Nicely done, thank you.